File: app/src/main/java/com/riders/thelab/data/remote/dto/Artist.java
package com.riders.thelab.data.remote.dto;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;
@Setter
@Getter
@ToString
@Parcel
public class Artist {
    @Json(name = "artistName")
    private String artistName;
    @Json(name = "firstName")
    private String firstName;
    @Json(name = "secondName")
    private String secondName;
    @Json(name = "lastName")
    private String lastName;
    @Json(name = "dateOfBirth")
    private String dateOfBirth;
    @Json(name = "origin")
    private String origin;
    @Json(name = "debutes")
    private String debutes;
    @Json(name = "activities")
    private String activities;
    @Json(name = "urlThumb")
    private String urlThumb;
    @Json(name = "description")
    private String description;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/music/Artist.kt
package com.riders.thelab.data.remote.dto.music

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Artist(
    @Json(name = "id")
    val id: Int,
    @Json(name = "name")
    val name: String
)
